packages/core/strapi/lib/commands/utils/commander.js

```diff
@@ -116,7 +117,7 @@ const confirmMessage = (message) => {
       },
     ]);
     if (!answers.confirm) {
-      exitWith(0);
+      exitWith(1, failMessage);
```

Comment:
Should we add a default value for the fail message?

Reply:
This is intended as a generic way to confirm a prompt, I'm not sure we could come up with something that wouldn't be more annoying to have to override every time. I'm already not completely convinced on this being an error exit code, because if the user is specifically saying "no" it's not an error, it's working as intended. (and I at least want to keep the link between "red message" and "error" and not deviate from that)

Reply:
> I'm already not completely convinced on this being an error exit code
Yeah, this is a tricky one.
Here are some examples about exit codes for canceled commands
![image](https://user-images.githubusercontent.com/25851739/223466599-082184f3-06fb-43a5-b149-a262af49fbce.png)

Reply:
Not sure which one we should use though.

Reply:
If I'm interpreting this right:
First example you cancelled during a prompt and got a 0.
Second example you cancelled while it was working and got a 1.
Third example was cancelled during a password prompt for sudo and got a 1.

I think my preference is for a 0/warning, but I'm not opposed to a 1/error, so I think we should just go with it since it makes it easier for anything reading the exit code to interpret it as "ok, there was an error so the transfer didn't succeed" and also lets the user know the same with red text.